exporters/variables-scss/src/content/generators/fileGenerator.ts

```diff
@@ -0,0 +1,56 @@
+import { TokenGroup, Token, TokenType } from '@supernovaio/sdk-exporters';
+import { generateFileContent } from './contentGenerator';
+
+const filesData = [
+  {
+    fileName: '_spacing.scss',
+    tokenTypes: TokenType.dimension,
+    groupNames: 'Spacing',
+    withCssObject: true,
+    hasParentPrefix: false,
+  },
+  {
+    fileName: '_radii.scss',
+    tokenTypes: TokenType.dimension,
+    groupNames: 'Radius',
+    withCssObject: true,
+    hasParentPrefix: false,
+  },
+  {
+    fileName: '_borders.scss',
+    tokenTypes: TokenType.dimension,
+    groupNames: 'Border',
+    withCssObject: false,
+    hasParentPrefix: true,
+  },
+  {
+    fileName: '_other.scss',
+    tokenTypes: [TokenType.dimension, TokenType.string],
+    groupNames: ['Grid', 'Container', 'Breakpoint'],
+    withCssObject: true,
+    hasParentPrefix: true,
+  },
+];
+
+export const generateFiles = (
+  tokens: Array<Token>,
+  mappedTokens: Map<string, Token>,
+  tokenGroups: Array<TokenGroup>,
+) => {
```

Comment:
```suggestion
) => filesData.map
```

You can use implicit return in this case.

Reply:
With this solutions I get the ESLint error "ESLint: Expected no linebreak before this expression. (implicit-arrow-linebreak)"

Reply:
Ok.